src/functionalTest/resources/features/F-001/F-001.feature

```diff
@@ -0,0 +1,161 @@
+    When  a request is prepared with appropriate values
+    And   the request [for retrieval of retrieval doc-store metadata for non-existing document id]
+    And   it is submitted to call the [Retrieval of metadata from doc-store for given document id] operation of [CCD data store]
+    Then  a negative response is received
+    And   the response [contains an HTTP 404 'Not Found']
+    And   the response has all the details as expected
+
+  @S-010
+  Scenario: Receive an error response, for retrieval of doc-store metadata for document id which is not associated
+    Given a user with [an active profile in CCD]
+    And   a user with [an active caseworker profile in CCD with full permission on specific document field]
+    And   a successful call [by a privileged user to upload a document with mandatory metadata] as in [Default_Document_Upload_Data]
+    And   a user with [an active citizen profile in CCD with full permission on specific document field]
+    And   a successful call [by a privileged user to upload a document with mandatory metadata] as in [Default_Document_Upload_Data]
+    When  a request is prepared with appropriate values
+    And   the request [for retrieval of doc-store metadata for document id uploaded by citizen]
+    And   it is submitted to call the [Retrieval of metadata from doc-store for given document id] operation of [CCD data store]
+    Then  a negative response is received
+    And   the response [contains an HTTP 403 'Forbidden']
+    And   the response has all the details as expected
+
+  @S-011
+  Scenario: Receive an error response, for retrieval of doc-store metadata for malformed document id
+    Given a user with [an active profile in CCD]
+    And   a user with [an active caseworker profile in CCD with full permission on specific document field]
+    And   a successful call [by a privileged user to upload a document with mandatory metadata] as in [Default_Document_Upload_Data]
+    When  a request is prepared with appropriate values
+    And   the request [for retrieval of doc-store metadata for malformed document id]
+    And   it is submitted to call the [Retrieval of metadata from doc-store for given document id] operation of [CCD data store]
+    Then  a negative response is received
+    And   the response [contains an HTTP 400 'Not Found']
+    And   the response has all the details as expected
+
+  @S-012
+  Scenario: must return a negative response when the request body doesn't provide a mandatory field document id
+    Given a user with [an active profile in CCD]
+    And   a user with [an active caseworker profile in CCD with full permission on specific document field]
+    And   a successful call [by a privileged user to upload a document with mandatory metadata] as in [Default_Document_Upload_Data]
+    When  a request is prepared with appropriate values
+    And   the request [body doesn't provide a mandatory field document id ]
```

Comment:
the space between `id` and `]` could cause some problems

Reply:
removed